clang/include/clang/Basic/DiagnosticDriverKinds.td

```diff
@@ -577,6 +577,16 @@ def err_drv_reduced_module_output_overrided : Warning<
   "please consider use '-fmodule-output=' to specify the output file for reduced BMI explicitly">,
   InGroup<DiagGroup<"reduced-bmi-output-overrided">>;
 
+def remark_found_cxx20_module_usage : Remark<
+  "found C++20 module usage in file '%0'">,
+  InGroup<ModulesDriver>;
+def remark_performing_explicit_module_build : Remark<
```

Comment:
I don't like the name "explicit module". It is confusing to me. Please change this to other names. e.g., "clang managed C++ modules build"